monitoring/alerts/updatealerts.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+if [ -z "$1" ]
```

Comment:
Is it possible to check on the proper argument values as specified in line 8?

Something like this would check if the variable is empty and it will check proper arg values:

```bash
if [[ -z "$1" ]] || grep -vqE 'pre|dev' <<< $1; then
    echo ""No target evironment argument supplied. Please provide target environment (e.g. pre, dev)"
    exit 1
fi
```